cognite/client/_api/files.py

```diff
@@ -709,18 +708,16 @@ def _upload_multipart_part(
         Args:
             upload_url (str): URL to upload file chunk to.
             content (str | bytes | TextIO | BinaryIO): The content to upload.
-            mime_type (str | None): Optional mime type for the `Content-Type` header.
         """
         if isinstance(content, str):
             content = content.encode("utf-8")
 
-        headers = {"Content-Type": mime_type}
         upload_response = self._http_client_with_retry.request(
             "PUT",
             upload_url,
             data=content,
             timeout=self._config.file_transfer_timeout,
-            headers=headers,
+            headers=None,
```

Comment:
I think `headers=None` is the default, you can probably remove this line completely.

Reply:
I prefer to keep it as it is then expressing that this request MUST not have any headers.

Reply:
...then you should add a comment